src/dataflow-types/src/client.rs

```diff
@@ -45,6 +50,11 @@ pub enum Command {
     doc = "The kind of compute command that was received"
 )]
 pub enum ComputeCommand {
+    /// Indicates the creation of an instance, and is the first command.
+    CreateInstance,
+    /// Indicates the termination of an instance, and is the last command.
+    DropInstance,
```

Comment:
I'm not sure I follow why these must be the first and last commands, respectively—unless you mean first and last command for a given ComputeInstanceID?

Reply:
That's right. Each of these is (in my mind, maybe not clearly enough) under a `ComputeInstanceId` and the comment was meant to be scoped by that too. I can improve the text, as long as the concept makes sense. The code does not currently validate any of these constraints.

Reply:
Definitely was confused by the scoping, but also I don’t have any improvements to suggest. LGTM!